custom_components/adaptive_lighting/switch.py

```diff
@@ -1078,24 +1148,37 @@ async def _adapt_light(
             color_temp_kelvin = max(min(color_temp_kelvin, max_kelvin), min_kelvin)
             service_data[ATTR_COLOR_TEMP_KELVIN] = color_temp_kelvin
         elif "color" in features and adapt_color:
-            _LOGGER.debug("%s: Setting rgb_color of light %s", self._name, light)
-            service_data[ATTR_RGB_COLOR] = self._settings["rgb_color"]
+            rgb_color = self._settings["rgb_color"]
+            _LOGGER.debug(
+                "%s: Setting rgb_color of light %s to ", self._name, light, rgb_color
+            )
+            service_data[ATTR_RGB_COLOR] = rgb_color
```

Comment:
This log message only contains 2 `%s` markers. So it is not really logging anything. Since this was likely only used for debugging, can you revert it to the original?